libpod/container_internal_linux.go

```diff
@@ -336,8 +336,9 @@ func (c *Container) generateSpec(ctx context.Context) (*spec.Spec, error) {
 	}
 
 	g.SetRootPath(c.state.Mountpoint)
-	g.AddAnnotation(crioAnnotations.Created, c.config.CreatedTime.Format(time.RFC3339Nano))
+	g.AddAnnotation(annotations.Created, c.config.CreatedTime.Format(time.RFC3339Nano))
 	g.AddAnnotation("org.opencontainers.image.stopSignal", fmt.Sprintf("%d", c.config.StopSignal))
+	g.AddAnnotation(annotations.ContainerManager, annotations.ContainerManagerLibpod)
```

Comment:
Can we check if an annotation with this key already exists and only add if one doesn't?

Reply:
Yes sure, I will change it this way.